uint16 	class_id
float32 score
rect rt
point cen_pt